### PR TITLE
[TIMOB-17729] Android: Add CanceledOnTouchOutside property to Titanium.U...

### PR DESCRIPTION
**Pull request by ashcoding**
...I.Android.ProgressIndicator

Jira: https://jira.appcelerator.org/browse/TIMOB-17729

### CODE REVIEW

**Comment by ashcoding** [comment]
```
Ti.UI.backgroundColor = 'white';
 
var win = Ti.UI.createWindow({
	backgroundColor : 'blue'
});
 
var button = Ti.UI.createButton({
	title : 'Show Progress Dialog'
});
 
var progressIndicator = Ti.UI.Android.createProgressIndicator({
	message : 'Loading...',
	location : Ti.UI.Android.PROGRESS_INDICATOR_DIALOG,
	type : Ti.UI.Android.PROGRESS_INDICATOR_DETERMINANT,
	cancelable : true,
	canceledOnTouchOutside : false,
	min : 0,
	max : 10,
	value : 0
});
 
button.addEventListener('click', function(e) {
	progressIndicator.show();
	var value = 0;
	setInterval(function() {
		if (value > 10) {
			return;
		}
		progressIndicator.value = value;
		value++;
	}, 200);
 
});
 
win.add(button);
win.open();
```

File: android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIProgressIndicator.java [hunk 1]
```diff
@@ -120,6 +120,11 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 				progressDialog.setCancelable(TiConvert.toBoolean(newValue));
 			}
 
+		} else if (key.equals(TiC.PROPERTY_CANCELED_ON_TOUCH_OUTSIDE)) {
+			if (progressDialog != null) {
```
Comment (hieupham007) [inline android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIProgressIndicator.java]: we can merge these ifs

File: apidoc/Titanium/UI/Android/ProgressIndicator.yml [hunk 2]
```diff
@@ -42,6 +42,13 @@ properties:
     type: Boolean
     platforms: [android]
 
+  - name: canceledOnTouchOutside
+    summary: |
+        Sets whether this dialog is canceled when touched outside the window's bounds.
+        Default is `true`.
+    type: Boolean
+    platforms: [android]
```
Comment (hieupham007) [inline apidoc/Titanium/UI/Android/ProgressIndicator.yml]: missing since?

**Comment by ashcoding** [comment]
Addressed comments.

**Comment by hieupham007** [comment]
Looks good. Merging

**Pull request merged**